xmtp_mls/migrations/2024-10-17-173334_create_v2_conversations/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE v2_conversations (
+    topic TEXT NOT NULL PRIMARY KEY,
+    created_at_ns BIGINT NOT NULL,
+    peer_address TEXT NOT NULL,
+    envelope_bytes BLOB NOT NULL
```

Comment:
The most expensive thing about listing v2 conversations is decrypting this envelope.

What if we actually stored the plaintext here so it only has to happen once?